source/guide_mailman-3.rst

```diff
@@ -225,13 +227,6 @@ After the REST backend has been configured, we need to configure the Django fron
  
  [...]
 
- USE_X_FORWARDED_HOST = True # Uncomment
- 
- SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https') # Uncomment
- SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_SCHEME', 'https') # Uncomment
```

Comment:
While the host header is correct when using web backends, the protocol is still HTTP, which is normal in most proxy setups. Are you sure removing this doesn't cause unwanted HTTP=>HTTPS redirect-loops from the web app?

Reply:
No, I'm not sure. I've tried to access the website after removing and had no issues. However I don't know how Django will react here exactly.

Reply:
I vote for leaving it in, but I'm fine with removing it, if it works without it.

Reply:
+1 for leaving it in as well as it is resembling the actual technical setup.